lib/LANraragi/Plugin/Metadata/RegexParse.pm

```diff
@@ -23,34 +74,49 @@ sub plugin_info {
         type        => "metadata",
         namespace   => "regexplugin",
         author      => "Difegue",
-        version     => "1.0",
+        version     => "1.0.1",
         description =>
           "Derive tags from the filename of the given archive. <br>Follows the doujinshi naming standard (Release) [Artist] TITLE (Series) [Language].",
         icon =>
           "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IArs4c6QAAAL1JREFUOI1jZMABpNbH/sclx8DAwPAscDEjNnEMQUIGETIYhUOqYdgMhTPINQzdUEZqGIZsKBM1DEIGTOiuexqwCKdidDl0vtT62P9kuZCJEWuKYWBgYGBgRHbh04BFDNIb4jAUbbSrZTARUkURg6lD10OUC/0PNaMYgs1Skgwk1jCSDCQWoBg46dYmhite0+D8pwGLCMY6uotRDOy8toZBkI2HIhcO/pxCm8KBUkOxFl/kGoq3gCXFYFxVAACeoU/8xSNybwAAAABJRU5ErkJggg==",
-        parameters => [ ]
+        parameters => [
+            { type => "bool", desc => "Capture trailing tags in curly brackets" },
```

Comment:
I would give an example of the naming scheme caught by this either here or in the main plugin description, it's a bit confusing out of the box otherwise.

Reply:
I also think this one would make sense to default to `true`.

Reply:
I thought about leaving it like this to keep the old behavior, but if you are sure I'll change the default.

About the description I'll see what I can do. This is the hardest part :rofl: